Admin/CategoryAdmin.php

```diff
@@ -19,7 +19,6 @@
 class CategoryAdmin extends ContextAwareAdmin
 {
     protected $formOptions = array(
-        'cascade_validation' => true,
```

Comment:
Found the answer myself : https://github.com/webmozart/symfony/blob/d85c8173bbc2ff49186d3be7ed9e4c00c3a18bf9/UPGRADE-2.6.md#form It is deprecated since 2.6